android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileFragment.kt

```diff
@@ -101,6 +110,12 @@ class ProfileFragment : Fragment() {
     }
   }
 
+  private suspend fun handleRefreshLiveData() {
+    with(profileFragmentArgs) {
+      profileViewModel.retrieveProfileUiState(profileId, resourceId, resourceConfig, params)
+    }
+  }
+
```

Comment:
I think we can move the function block (and delete this private function) to where this is called as this private function is only called from one place.